packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet.tsx

```diff
@@ -227,7 +182,15 @@ export class AtomicSmartSnippet implements InitializableComponent {
           {this.renderQuestion()}
           {this.renderContent()}
           <footer part="footer">
-            {this.renderSource()}
+            {this.smartSnippetState.source ? (
+              <atomic-smart-snippet-source
+                source={this.smartSnippetState.source}
+                smartSnippet={this.smartSnippet}
+                isSuggestion={false}
+              ></atomic-smart-snippet-source>
+            ) : (
+              []
+            )}
```

Comment:
IMO this should stay inside a `renderSource` function. I don't feel strongly about it though.